src/Drupal/Commands/core/MigrateRunnerCommands.php

```diff
@@ -759,4 +757,28 @@ protected function getSourceIdKeys(MigrateIdMapInterface $idMap): array
         }, array_keys($columns));
         return array_combine($sourceIdKeys, $sourceIdKeys);
     }
+
+    /**
+     * Validate that a migration name is valid.
+     *
+     * If the argument to be validated is not named migrationId, pass the
+     * argument name as the value of the annotation.
+     *
+     * @hook validate @validate-migration-name
+     *
+     * @param \Consolidation\AnnotatedCommand\CommandData $commandData
+     *
+     * @return \Consolidation\AnnotatedCommand\CommandError|null
+     */
+    public function validateMigrationName(CommandData $commandData)
+    {
+        $arg_name = $commandData->annotationData()->get('validate-migration-name', null) ?: 'migrationId';
```

Comment:
The `null` default value can be removed. If missing a default value, an empty string is returned and the whole line work in the same way